corehq/apps/app_manager/static/app_manager/js/manage_releases.js

```diff
@@ -0,0 +1,104 @@
+hqDefine('app_manager/js/manage_releases', [
+    'jquery',
+    'knockout',
+    'underscore',
+    'hqwebapp/js/initial_page_data',
+    'hqwebapp/js/widgets_v4',
```

Comment:
nit: we have an unwritten convention to add a comment stating what widget you are using from this import, would be useful to add here.